.ci/test.sh

```diff
@@ -150,30 +155,31 @@ if [[ $TASK == "check-docs" ]] || [[ $TASK == "check-links" ]]; then
 fi
 
 if [[ $PYTHON_VERSION == "3.7" ]]; then
-    CONDA_REQUIREMENT_FILES="--file ${BUILD_DIRECTORY}/.ci/conda-envs/ci-core-py37.txt"
+    CONDA_REQUIREMENT_FILES="${BUILD_DIRECTORY}/.ci/conda-envs/ci-core-py37.txt"
 elif [[ $PYTHON_VERSION == "3.8" ]]; then
-    CONDA_REQUIREMENT_FILES="--file ${BUILD_DIRECTORY}/.ci/conda-envs/ci-core-py38.txt"
+    CONDA_REQUIREMENT_FILES="${BUILD_DIRECTORY}/.ci/conda-envs/ci-core-py38.txt"
 else
-    CONDA_REQUIREMENT_FILES="--file ${BUILD_DIRECTORY}/.ci/conda-envs/ci-core.txt"
+    CONDA_REQUIREMENT_FILES="${BUILD_DIRECTORY}/.ci/conda-envs/ci-core.txt"
```

Comment:
If we're going to move `--file` out of the string, then please also rename the variable from `CONDA_REQUIREMENT_FILES` (plural) to `CONDA_REQUIREMENT_FILE` (singular).

I'd intentionally named it `*_FILES` with so that it could be extended with additional requirement files in the future, for example if we wanted to constrain some dependencies.

But for now, since that isn't happening, I'm fine with removing the repeated `--files` ... if the variable name is changed as well.

Reply:
Looks like this was resolved by this commit: https://github.com/microsoft/LightGBM/pull/6646/commits/a7222103eabee13c4d1fa9ce54befc6b3d61d31c

Please, in the future, post a comment like "fixed in https://github.com/microsoft/LightGBM/pull/6646/commits/a7222103eabee13c4d1fa9ce54befc6b3d61d31c" in these threads and then click "Resolve conversation". That helps reviewers understand what is still remaining to be done.